### ISSUE TITLE
Get the mean and standard deviation of a polynomial chaos

### ISSUE THREAD

**Issue by mbaudin47** [openturns/openturns#1927]
Once the coefficients of a polynomial chaos have been estimated, we may be interested by its mean and standard deviation.
The current interface does not provide a way to access these results. 
The only way is currently to use `FunctionalChaosSobolIndices`, which has a handy `summary` method:
```python
chaosSI = ot.FunctionalChaosSobolIndices(result)
print(chaosSI.summary())
 input dimension: 4
 output dimension: 1
 basis size: 25
 mean: [12.5763]
 std-dev: [4.18011]
[...]
```
Still, the code is internally available [from the summary method](https://github.com/openturns/openturns/blob/b5797d7e4a71c71faf86df51f26ad0d8d551ad08/lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosSobolIndices.cxx#L68).
As a workaround, we may use the following Python code, adapted from C++.
```python
def get_PC_mean_sd(functionalChaosResult):
    """
    Return mean and standard deviation of a polynomial chaos.

    This code was adapted from 
    https://github.com/openturns/openturns/blob/b5797d7e4a71c71faf86df51f26ad0d8d551ad08/lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosSobolIndices.cxx#L68
    
    Parameters
    ----------
    functionalChaosResult : ot.FunctionalChaosResult()
        The polynomial chaos.

    Returns
    -------
    mean : ot.Point(outputDimension)
        The expectation of Y = p(X), where p is the polynomial.
    stdDev : ot.Point(outputDimension)
        The standard deviation of Y = pc(X), where pc is the polynomial chaos,
        as the square root of the variance.
    """
    outputDimension = functionalChaosResult.getMetaModel().getOutputDimension()
    indices = functionalChaosResult.getIndices()
    coefficients = functionalChaosResult.getCoefficients()
    basisSize = indices.getSize()
    variance = ot.Point(outputDimension)
    mean = ot.Point(outputDimension)
    for i in range(outputDimension):
        for k in range(basisSize):
            # Take into account only non-zero indices as the
            # null index is the mean of the vector
            if indices[k] > 0:
                variance[i] += coefficients[k, i] ** 2
            else:
                mean[i] = coefficients[k, i]
    # standard deviation
    stdDev = ot.Point(outputDimension)
    for i in range(outputDimension):
        stdDev[i] = np.sqrt(variance[i])
    return mean, stdDev
```
A solution would be to provide two new `FunctionalChaosSobolIndices.getMean()` and `FunctionalChaosSobolIndices.getStandardDeviation()` methods. 

Please do not implement that right away: it may be included in a next contract within a few weeks.

**Comment by regislebrun** [comment]
@mbaudin47 Please have a look at FunctionalChaosRandomVector:
```
import openturns as ot
x = ot.Normal(2).getSample(10000000)
y = ot.SymbolicFunction(["x1", "x2"], ["1+sin(x1)*(1+x2)"])(x)
print("y mean=", y.computeMean())
print("y cov=", y.computeCovariance())
x = ot.Normal(2).getSample(1000)
y = ot.SymbolicFunction(["x1", "x2"], ["1+sin(x1)*(1+x2)"])(x)
algo = ot.FunctionalChaosAlgorithm(x, y)
algo.run()
rv = ot.FunctionalChaosRandomVector(algo.getResult())
print("rv mean=", rv.getMean())
print("rv cov=", rv.getCovariance())
```
gives you:
```
y mean= [0.999936]
y cov= [[ 0.86505 ]]
rv mean= [0.985993]
rv cov= [[ 0.815758 ]]
```

**Comment by mbaudin47** [comment]
Oups: my bad. Thank you for the tip. And FunctionalChaosSobolIndices cannot use FunctionalChaosRandomVector because both are  at the same level, on top of FunctionalChaosResult.
This an invalid ticket.

**Comment by regislebrun** [comment]
You can create a FunctionalChaosRandomVector from within FunctionalChaosSobolIndices and get the mean and the covariance without trouble, you just have to pass the result given to the constructor of FunctionalChaosSobolIndices to the constructor of FunctionalChaosRandomVector!

**Comment by mdelozzo** [comment]
I didn't find how to get PCE mean and variance and so, I coded something in my corner this morning, based on ``FunctionalChaosResult.getCoefficients()``. Then, I talked with a colleague who had the same problem a few weeks ago and used the same kind of Python lines. I then searched the open and closed issues related to this topic and found this one.

Why not adding ``FunctionalChaosResult.getMean()`` and ``FunctionalChaosResult.getMean()``? Or at least advertise ``FunctionalChaosRandomVector`` in the documentation of ``FunctionalChaosResult``. 

We could also imagine an example "Get statistics from polynomial chaos" or "Get mean and variance from polynomial chaos".
